crates/net/network/tests/it/multiplex.rs

```diff
@@ -22,7 +22,7 @@ use std::{
 use tokio::sync::{mpsc, oneshot};
 use tokio_stream::wrappers::UnboundedReceiverStream;
 
-/// A simple Rplx subprotocol for
+/// A simple Rlpx subprotocol for
```

Comment:
@mattsse for ..? 😄

Reply:
> @mattsse for ..? 😄

I can complete it if you tell me what to do

Reply:
```suggestion
/// A simple Rlpx subprotocol that sends pings and pongs
```